doc/new-guidelines.md

```diff
@@ -313,6 +338,18 @@ memory.
  "Markov"]
 ```
 
+Avoid dynamic labels, for example to specify an element's index because
+[Appium](https://appium.io/) already supports element selection based on
+indices.
+
```

Comment:
Oh I never knew that

Reply:
I didn't know either, but the QA team recently confirmed this.